css/properties/vector-effect.json

```
@@ -25,7 +25,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤5.1"
```

```suggestion
              "version_added": "5.1"
```

See: https://github.com/caugner/css-properties/commit/fbada1447b1d3b57575db1681cb1c37afa477e5a#diff-b63d72471087ee495d4f5c7e94b58bef200825135bdee1bdcddbfbd594b0c328R223